google-cloud-firestore/samples/acceptance/query_watch_test.rb

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "helper.rb"
+require_relative "../query_watch.rb"
+
+describe "Google Cloud Firestore API samples - Query Data" do
+  before :all do
+    @firestore_project = ENV["FIRESTORE_PROJECT"]
```

Comment:
Curious why this has to be in a `before_all` hook, as it's not expensive or anything. (I'm also surprised that setting an instance variable actually works in a `before_all` hook, because how could it set the variable for each individual test? But I guess if it's working then...)

Reply:
It can be inlined. This was inherited from ruby-docs-samples.

Reply:
It can be in a `before` block, just doesn't seem like it should be in `before_all`.

Reply:
Ah, gotcha. It just wound up being the only thing left in there as I moved other setup out.

Reply:
Moved to `before`.